desktop-environment.el

```diff
@@ -357,22 +361,36 @@ portion of the screen."
 ;;; Commands - screenshots
 
 ;;;###autoload
-(defun desktop-environment-screenshot ()
+(defun desktop-environment-screenshot (delay)
   "Take a screenshot of the screen in the current working directory."
```

Comment:
can you please add a reference to `DELAY` in the docstring as well as a reference to `desktop-environment-screenshot-delay-argument`?

Reply:
Should be resolved as of 0e0d97a